platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateFileManager.java

```diff
@@ -97,6 +99,11 @@ public class SignedStateFileManager implements Startable {
      */
     private final Time time;
 
+    /**
+     * Manages platform status
+     */
+    private final PlatformStatusComponent platformStatusComponent;
```

Comment:
Instead of having the entire `PlatformStatusComponent` being passed around everywhere, maybe it's better to pass around a functional interface for submitting `PlatformStatusAction`. None of these classes should know about or be able to invoke the `start()` and `stop()` methods.

Reply:
Agreed, done